airflow/config_templates/config.yml

```diff
@@ -1631,6 +1640,24 @@
       type: string
       example: ~
       default: "True"
+- name: aws_ssm_creds
```

Comment:
I understand that every credentials provider might have their own specific configuration. However maybe we can have a way to find a more generic configuration approach? I would rather register the credentials provider as a plugin and configure it same way as plugins are configured. 

I do not know how yet - but maybe we can find a way.

Again - this is not a very strong preference, this is rather "thinking out loud".

Reply:
@potiuk i have consolidated all configuration related to creds backend under new section `[creds_backend]`.
currently there are 3 params:
```
class_list
aws_ssm_prefix
aws_ssm_profile_name
```
so now it the suggestion to contributors would be if there are new backends, just add the implementation-specific params under `[creds_backend]` with appropriate prefix.

let me know if you see opportunity for further improvement.